api/v1/types/model_types.go

```diff
@@ -31,3 +31,27 @@ type MetricResult struct {
 type MetricResultList struct {
 	Items []MetricResult `json:"items"`
 }
+
+type Stats struct {
+	Average    uint64 `json:"average"`
+	Percentile uint64 `json:"percentile"`
```

Comment:
95th. I'll change that name, as well as the Stats struct in model/types.go
However, I will not change the json representation as of now to avoid conflicts in Kubedash